pom.xml

```diff
@@ -301,12 +298,20 @@
                         <module>integration_tests</module>
                         <module>udf_coverage</module>
                     </modules>
+                    <linkReplacements>
+                        <linkReplacement>
+                            https://github.com/exasol/test-db-builder|https://github.com/exasol/test-db-builder-java
+                        </linkReplacement>
+                        <linkReplacement>
+                            https://github.com/exasol/udf-debugging-javat|https://github.com/exasol/udf-debugging-java
```

Comment:
```suggestion
                            https://github.com/exasol/udf-debugging-java|https://github.com/exasol/udf-debugging-java
```

Reply:
They are duplicated with with separator, is it required?

Reply:
It's for replacing broken links. <broken link>|<replacment>
It should be as it is now